.vscode/settings.json

```diff
@@ -1,9 +1,14 @@
 {
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#4459ff",
-    "titleBar.activeForeground": "#fff",
-    "titleBar.inactiveBackground": "#4459ff1a",
-    "titleBar.inactiveForeground": "#858b9a",
-    "activityBar.activeBorder": "#4459ff"
-  }
+  "tailwindCSS.experimental.configFile": {
+    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
+    "packages/design-system-twrnc-preset/tailwind.config.js": "packages/design-system-react-native/src/**"
+  },
+  "files.associations": {
+    "*.css": "tailwindcss"
+  },
+  "editor.quickSuggestions": {
+    "strings": true
+  },
+  "tailwindCSS.classFunctions": ["tw", "tw`", "twClassName"],
```

Comment:
Tells the extension where to provide Tailwind class name completions in JavaScript/TypeScript code.
- "tw" - Enables IntelliSense for TWRNC function calls
- "tw`" - Enables IntelliSense for TWRNC template literals
- "twClassName" - Enables IntelliSense when using this prop